playground/DatabaseMigration/DatabaseMigration.ApiService/Directory.Build.props

```diff
@@ -0,0 +1,11 @@
+<Project>
+
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
+
+  <PropertyGroup>
+    <!-- Clear solution default because EF tooling doesn't like a custom output path. -->
+    <!-- Issue: https://github.com/dotnet/efcore/issues/23853 -->
+    <BaseIntermediateOutputPath></BaseIntermediateOutputPath>
+  </PropertyGroup>
```

Comment:
Can we just do this in a `Directory.Build.props` in the `playground\DatabaseMigration` folder instead of having this duplicated across all the new projects?

Reply:
I tried doing that, but it introduced a new problem with the Aspire project not able to find its props/targets.

BaseIntermediateOutputPath change only needs to be done inside the database migration example.

Reply:
>  but it introduced a new problem with the Aspire project not able to find its props/targets.

I don't understand what this means. Can you elaborate?

> BaseIntermediateOutputPath change only needs to be done inside the database migration example.

Right, that's why I suggest adding it to the playground\DatabaseMigration folder.

Reply:
Now it works https://github.com/dotnet/aspire/pull/2327 🤷